src/zkevm_specs/util/gas.py

```diff
@@ -0,0 +1,43 @@
+from ..encoding import U8, U64, U128, U256
```

Comment:
I think we should rename this file to `testing.py` to reduce the confusion as it only provides utility functions for testing.

Reply:
make sense , will update !

Reply:
updated in [0912af8](https://github.com/appliedzkp/zkevm-specs/pull/125/commits/0912af8352fad807c9c70e00c0cd89c1e7532ded)